core/simulations/index.ts

```diff
@@ -0,0 +1,44 @@
+import prompts, { PromptObject } from 'prompts';
+import { SIMULATIONS } from './config';
+import readline from 'readline';
+
+const keypress = async () => {
+    await prompts({
+        type: 'invisible',
+        name: 'irrelevant',
+        message: 'Press Enter to continue',
+    });
+};
+
+const selectAndRunSimulation = async () => {
+    readline.emitKeypressEvents(process.stdin);
+    const promtChoices = SIMULATIONS.map(simulationConfig => simulationConfig.title).map(simulationName => ({
+        title: simulationName,
+        value: simulationName,
+    }));
+    const promtConfig: PromptObject = {
+        type: 'select',
+        name: 'value',
+        message: 'Select Simulation',
+        choices: promtChoices,
+    };
+    while (true) {
+        const answer = await prompts(promtConfig);
+        const simulationConfig = SIMULATIONS.find(item => item.title === answer.value);
+        if (!simulationConfig) {
+            throw new Error(`Simulation config not found: ${answer.value}`);
+        }
+        for (const [i, step] of simulationConfig.steps.entries()) {
+            console.info(`Next step is "${step.title}"`);
+            if (i !== 0) {
+                await keypress();
+            }
```

Comment:
```suggestion
            if (i !== 0) {
                await keypress(step.title);
            }
```

Then please also update `keypress` to `Press enter to proceed to the next step "${stepTitle}"` to be more concise and print less lines as suggested in the https://github.com/sidestream-tech/unified-auctions-ui/issues/420#issuecomment-1225615981